File: 프로그래머스/0/120830. 양꼬치/양꼬치.py
def solution(n, k):
    yang = 12000 # 양꼬치
    drink = 2000 # 음료
    yang_price = 12000 * n # 양꼬치 주문 가격
    drink_price = 2000 * k # 음료 주문 가격
    service = (n//10) * 2000 # 서비스 가격
    
    # 총 지불 가격
    tot_price = yang_price + drink_price - service
    return tot_price

File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int yang = 12000; // 양꼬치
        int drink = 2000; // 음료
        
        // 양꼬치 + 음료 총합
        int yang_price = yang * n;
        int drink_price = drink * k;
        
        // 양꼬치 10인분에 음료 +1
        int service = (n/10)*2000;
        int tot_price = yang_price + drink_price - service;
        
        return tot_price;
    }
}